pkg/generate/dockerfile/parser.go

```diff
@@ -0,0 +1,104 @@
+package dockerfile
+
+import (
+	"bufio"
+	"fmt"
+	"io"
+	"regexp"
+	"strings"
+	"unicode"
+)
+
+type dockerfile [][]string
+
+// Parser is a Dockerfile parser
+type Parser interface {
```

Comment:
Yes, for maintainability going forward it makes sense. I will create an issue to do a follow up replacement. For what we need now, this parser is good enough.